tests/Bonami/Collection/EitherTest.php

```diff
@@ -238,6 +238,44 @@ public function testEach(): void
         self::assertEquals(42, $accumulator);
     }
 
+    public function testTap(): void
+    {
+        $right = Either::right(42);
+        $left = Either::left(666);
+        $accumulated = 0;
+
+        $accumulate = static function (int $i) use (&$accumulated): void {
+            $accumulated += $i;
```

Comment:
I would not like unexperienced devs to have impression, that this is the way, how to "fold" list, but they probably wont look at the test cases :-D

Reply:
I had similar thoughts. But I hope, that we are targeting on more experienced audience. Also, there are some more obvious methods for "folding", like `sum`, `reduce`, which are obvious enough (mfold is not so obvious for unexperienced audience).

Anyway, we need to improve docs to help developers with choosing best practices.